src/main/java/rs/teslaris/core/model/document/PersonDocumentContribution.java

```diff
@@ -8,13 +8,12 @@
 import javax.persistence.Table;
 import lombok.Getter;
 import lombok.Setter;
-import org.hibernate.annotations.Where;
 
 @Getter
 @Setter
 @Entity
 @Table(name = "person_document_contributions")
-@Where(clause = "deleted=false")
+//@Where(clause = "deleted=false")
```

Comment:
If a document is deleted, what will happen with PersonDocumentContribution? The same question for all other person contributions

Reply:
I have tested it, and unfortunately it does not but if we include the ennotation in the code we encounter another bug. After a few hours of trying to fix said problem I was unsuccessful. I will consult with @milospp and we will try to find a solution together.